dunwoody_disaster/views/victoryScreen.py

```diff
@@ -1,43 +1,229 @@
+
+        layout.addItem(
+            QSpacerItem(250, 0, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 2
+        )
+
+        lbl = QLabel("LOOT!")
+        lbl.setStyleSheet("font-size: 18px; color: white;")
+        lbl.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        layout.addWidget(lbl, row, 3)
+        row += 1
+
+        layout.addItem(QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 1)
+        row += 1
 
         pic = QLabel("")
         pic.setAlignment(Qt.AlignmentFlag.AlignCenter)
-        pic.setStyleSheet("min-width: 380px;")
+        pic.setStyleSheet("min-width: 300px;")
         pic.setPixmap(fight_controller.player.image().scaledToWidth(300))
-        layout.addWidget(pic)
+        layout.addWidget(pic, row, 1)
 
-        lbl = QLabel("Manage your inventory: ")
-        lbl.setAlignment(Qt.AlignmentFlag.AlignCenter)
-        layout.addWidget(lbl)
+        drops = QGroupBox("Loot Dropped")
+        drops.setStyleSheet("border: 1px solid red; color: white;")
+        loot = QHBoxLayout()
+        scroller = DD.scroller(loot, True, False)
+        scroller.setStyleSheet("border: none;")
+        drops.setLayout(DD.layout(scroller))
+        layout.addWidget(drops, row, 3)
+
+        for item in self.items:
+            widget, box = self.create_inventory_slot(item)
+            loot.addWidget(widget)
+            self.boxes[box] = item
+
+        row += 1
+
+        # layout.addItem(QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 1)
+        # row += 1
+
+        # lbl = QLabel("Manage your inventory: ")
+        # lbl.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        # layout.addWidget(lbl, row, 1)
+        # row += 1
+
+        layout.addItem(QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 1)
+        row += 1
+
+        cap_Layout = QGridLayout()
+        cap_Layout.setSpacing(0)
+        cap_Layout.setContentsMargins(0, 0, 0, 0)
+
+        lbl = QLabel("Inventory capacity: ")
+        lbl.setStyleSheet("font-size: 16px;")
+        cap_Layout.addWidget(lbl, 0, 0)
+
+        self.capacity = Meter(QColor("white"), 0)
+        self.capacity.setEndColor(QColor("red"))
+        self.capacity.setMinimumHeight(25)
+        cap_Layout.addWidget(self.capacity, 0, 1)
+        row += 1
+
+        layout.addLayout(cap_Layout, row, 1)
+        row += 1
+
+        layout.addItem(QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 1)
+        row += 1
+
+        inventory_box = QGroupBox("Inventory")
+        inventory_box.setStyleSheet("border: 1px solid red; color: white;")
+        inventory = QHBoxLayout()
+        scroller = DD.scroller(inventory, True, False)
+        scroller.setStyleSheet("border: none;")
+        inventory_box.setLayout(DD.layout(scroller))
+        layout.addWidget(inventory_box, row, 1)
+        row += 1
 
-        loot_screen = CollectLootScreen(
-            fight_controller.player, fight_controller.enemy.get_items()
```

Comment:
I feel like there was a better way this could have been done.

Reply:
This was the easiest way for reading/style/layout alterations. Also I needed to split up components to position individually.